Makefile

```diff
@@ -73,6 +76,73 @@ SHELL = /usr/bin/env bash -o pipefail
 
 all: build
 
+##@ Tools
+
+OPERATOR_SDK = $(PROJECT_PATH)/bin/operator-sdk
+OPERATOR_SDK_VERSION = v1.22.0
+$(OPERATOR_SDK):
+	./utils/install-operator-sdk.sh $(OPERATOR_SDK) $(OPERATOR_SDK_VERSION)
+
+.PHONY: operator-sdk
+operator-sdk: $(OPERATOR_SDK) ## Download operator-sdk locally if necessary.
+
+CONTROLLER_GEN = $(PROJECT_PATH)/bin/controller-gen
+$(CONTROLLER_GEN):
+	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.10.0)
```

Comment:
maybe a good opportunity to bump it to 0.11.x ?

Reply:
done